include/vfio-user.h

```diff
@@ -1,34 +1,29 @@
-/*-
- *   BSD LICENSE
- *
- *   Copyright (c) Intel Corporation.
- *   All rights reserved.
+/*
+ * Copyright (c) 2020 Intel Corporation. All rights reserved.
  *
```

Comment:
Should we also add Nutanix here? As we also made changes to this file and we will be maintaining it.

Reply:
Good point, updated.